sydneywait/Kanawaha-Trace
# Route details

Using waypoint information, I have added details to the route-details page that allows the user to see a description of certain waypoints between their selected start and end points.  It zooms into the map to show the extent of their trail selection, and populates markers on the map.  When the user hovers over these markers, it tells them the name of the waypoint and the mileage.

To test:

Log in and look at the routes page.  Click on "details".  The map should automatically render with blue markers for the start and end points, and green markers for the waypoint features in between.  If you edit the start and/or end point on the map, the map will re-render and show the new map segment.  Al other functionality should remain as previous tested.
